src/Components/RestPassword/ResetInterface.ts

```diff
@@ -0,0 +1,6 @@
+export interface Data{
```

Comment:
The name data is too generic, we can change it to something like ResetFormInterface.